### DISCUSSION TITLE
New design proposal

### DISCUSSION THREAD

**Discussion by Alystrasz** [0neGal/viper#18] [Ideas]
As stated [here](https://github.com/0neGal/viper/pull/8#issuecomment-1002800344), I started to work on a more traditional design for the app.

It will have all features of current GUI (of course), plus :
* release notes (for both Northstar and Viper)
* general Viper information (where to report issues mainly)
* credits regarding all used resources

**Comment by 0neGal** [comment]
Once I've finished the mod-support I'll work on a new design that's able to have a lot more content with a lot less problems... I already have some ideas, generally a page structured design, where the normal update/install/launch is always visible and to the side the mods, release notes and alike is there. And also maybe a settings tab...

Generally I have a good idea of what to do...

**Comment by Alystrasz** [comment]
Please focus on the mods support, I'll send some mockups as soon as I have something looking great.

**Comment by Alystrasz** [comment]
Let's discuss this on #21, since I think we should work on this before public release.
